src/randoop/main/GenTests.java

```diff
@@ -429,264 +399,335 @@ public boolean handle(String[] args) throws RandoopTextuiException {
         }
       }
     }
+    
+    ExecutionVisitor visitor;
+    if (visitors.isEmpty()) {
```

Comment:
I find this idiom (requiring exactly one, non-null ExecutionVisitor) clumsy, and I wonder why not have a list (that could be empty or even null) rather than having to have exactly one.  That could make it possible to eliminate both DummyVisitor and MultiVisitor from Randoop.  Maybe that is something to change at a future date.  Or maybe this idiom is cleaner for some reason?